index.js

```diff
@@ -77,7 +77,11 @@ extensions.forEach(function (language) {
 var getDeps = function (deps) {
   return Object.keys(deps).map(function (depname) {
     var dep = require(path.resolve(path.dirname(argv._[0])) + '/node_modules/' + depname + '/package.json')
-    dep.repository.url = 'http://ghub.io/' + depname
+    if (typeof (dep.repository) === 'string' || typeof (dep.repository) === 'undefined') {
+      dep.repository = { url: 'http://ghub.io/' + depname }
+    } else {
+      dep.repository.url = 'http://ghub.io/' + depname
```

Comment:
Why don't we abandon all these checks and just do:

```
dep.repository = 'https://ghub.io/' + depname
```

What I'm suggesting here is that we can disregard the existing value and just always use ghub.io

Reply:
Also, note that ghub supports HTTPS now. 🔒

Reply:
Sure no problem, just have to take care about it inside the template :+1: